BinaryObjectScanner/Protection/CopyX.cs

```diff
@@ -0,0 +1,107 @@
+        // I am assuming based on 0 evidence that the former is Light and the latter is Professional.
+        // Because I have no evidence, I am just returning copy-X for now. I have pre-emptively begun seperating the two,
+        // just for when a designation can be applied for sure.
+
+        // Overall:
+        // Whenever I say "at the end of" or "at the start of" pertaining to the filesystem, I mean alphabetically, because this is how copy-X images seem to be mastered.
+        // Both Light and Professional have a directory at the end of the image. The files within this directory are intersected by the physical ring.
+        // This file is usually called ZDAT, but not always. At least one instance of Light calls it ZDATA. At least one instance of Professional calls it System.
+        // Seemingly it can be anything. It doesn't help that most samples are specifically from one company's games, Tivola. Still, most use ZDAT.
+
+        //Professional:
+        //All instances of professional contain a disc check, performed via optgraph.dll. 
+        //All instances of professional contain in the directory at the end of the image 3 files. gov_[something].x64, iofile.x64, and sound.x64.
+        //Due to gov's minor name variance, sound.x64 sometimes being intersected by a ring at the start, and iofile.x64 being referenced directly in optgraph.x64, I chose to just check iofile.x64.
+        //It always starts the same, so if there are false positives, a more advanced check can be used if necessary.
+        // TODO: optgraph.dll also contains DRM to prevent kernel debugger SoftICE from being used, via a process called SoftICE-Test. I don't know if this is specifically part of copy-X, or if it's an external solution employed by both copy-X and also other companies. If it's the latter, it should have its own check. It has none here since it wouldn't be necessary.
+
+
+        /// <inheritdoc/>
+        public string? CheckPortableExecutable(string file, PortableExecutable pex, bool includeDebug)//Checks for Professional
+        {
+            //I don't need this to check optgraph.dll, that's redundant. Unsure how to exclude that.
+            if (pex.OverlayStrings != null)
+            {
+                if (pex.OverlayStrings.Any(s => s.Contains("optgraph.dll")))//Checks if main executable contains reference to optgraph.dll. Emergency 4's is missing this for some reason. 
+                    // TODO: TKKG also has an NE 3.1x executable with a reference. This can be added later.
+                    return "copy-X";
+            }
+
+            return null;
+        }
+
+        /// <inheritdoc/>
+        public string? CheckFilePath(string path)//Checks for Professional
+        {
+            if (Path.GetFileName(path).Equals("optgraph.dll", StringComparison.OrdinalIgnoreCase))//Filename check for optgraph.dll disc check
```

Comment:
Ditto to the PE checks, please add sources for existing checks (and comments, if relevant).

Reply:
Should be fixed, unless I've not formatted it how you want it.